lib/easypost/beta/referral.rb

```diff
@@ -107,4 +107,52 @@ def self.add_credit_card(referral_api_key, number, expiration_month, expiration_
     response = create_easypost_credit_card(referral_api_key, stripe_credit_card_token, priority)
     EasyPost::Util.convert_to_easypost_object(response, referral_api_key)
   end
+
+  # Add a Stripe payment method to a Referral Customer. This function requires the Referral Customer's API key.
+  # @param [String] stripe_customer_id Unique customer ID provided by Stripe.
+  # @param [String] payment_method_reference ID of the card or bank account provided by Stripe.
+  # @param [String] payment_method_type Which priority to save this payment method as on EasyPost, either 'primary' or 'secondary'.
+  # @param [String] api_key Override the API key used for this request.
+  # @return [EasyPost::PaymentMethod] The newly-added payment method.
+  # noinspection RubyParameterNamingConvention
+  def self.add_payment_method(stripe_customer_id, payment_method_reference, priority = 'primary', api_key = nil)
```

Comment:
```suggestion
  # @param [String] primary_or_secondary Which priority to save this payment method as on EasyPost, either 'primary' or 'secondary'.
  # @param [String] api_key Override the API key used for this request.
  # @return [EasyPost::PaymentMethod] The newly-added payment method.
  # noinspection RubyParameterNamingConvention
  def self.add_payment_method(stripe_customer_id, payment_method_reference, primary_or_secondary = 'primary', api_key = nil)
```

Reply:
Per other PRs, we should call this parameter `primary_or_secondary`

Reply:
"priority" is already used in other functions, so we'll keep it consistent and not introduce breaking changes for the existing function.s